doc/modules/model_evaluation.rst

```diff
@@ -1109,6 +1110,53 @@ function:
     >>> matthews_corrcoef(y_true, y_pred)  # doctest: +ELLIPSIS
     -0.33...
 
+.. _multilabel_confusion_matrix:
+
+Multi-label confusion matrix
+----------------
+
+The :func:`multilabel_confusion_matrix` function compute multi-label confusion
+matrix label-wisely (default) or sample-wisely (samplewise=True) to evaluate
+the accuracy of a classification.
+
+When calculating label-wise multi-label confusion matrix :math:`MCM`, the
+count of true negatives for class :math:`i` is :math:`MCM_{i,0,0}`, false
+negatives is :math:`MCM_{i,1,0}`, true positives is :math:`MCM_{i,1,1}`
+and false positives is :math:`MCM_{i,0,1}`. Here is an example::
+
+    >>> from sklearn.metrics import multilabel_confusion_matrix
+    >>> y_true = [[1, 0, 1], [0, 1, 0]]
+    >>> y_pred = [[1, 0, 0], [0, 1, 1]]
+    >>> multilabel_confusion_matrix(y_true, y_pred, samplewise=True)
```

Comment:
Can you evaluate the same data with samplewise=False?

Reply:
sure